DATA_FILE has passed deprecation period and should be removed

**Describe the bug**
ERT has gotten a tag (3.0.0rc0) higher than 2.43, and then a test fails:
```
    def test_run_template_replace_in_ecl_data_file(key, expected):
        """
        This test that we copy the DATA_FILE into the runpath,
        do substitutions and rename it from the DATA_FILE name
        to ECLBASE
        """
        if packaging.version.parse(version) > packaging.version.parse("2.43"):
>           pytest.fail("DATA_FILE has passed deprecation period and should be removed")
E           Failed: DATA_FILE has passed deprecation period and should be removed

/home/runner/work/ert/ert/tests/libres_tests/res/enkf/test_run_path_creation.py:171: Failed
```

**To reproduce**
Steps to reproduce the behaviour:
1. `pytest tests/libres_tests/res/enkf/test_run_path_creation.py `

**Expected behaviour**
No test failure

**Environment**
 - OS: [e.g. RHEL7]
 - ERT/Komodo release: [e.g. 2.18 or 2020.12.00]
 - Python version
 - Remote/HPC execution involved: [yes|no]

**Additional context**
Add any other context about the problem here.

--- a/src/ert/ensemble_evaluator/builder/_prefect_forkserver_preload.py
+++ b/src/ert/ensemble_evaluator/builder/_prefect_forkserver_preload.py
@@ -34,7 +34,7 @@ def _preload_scproxy() -> None:
     import requests  # pylint: disable=import-outside-toplevel
 
     try:
-        requests.get("http://localhost")
+        requests.get("http://localhost")  # pylint: disable=missing-timeout
     except requests.RequestException:
         pass
 

--- a/tests/libres_tests/res/enkf/test_run_path_creation.py
+++ b/tests/libres_tests/res/enkf/test_run_path_creation.py
@@ -167,7 +167,7 @@ def test_run_template_replace_in_ecl_data_file(key, expected):
     do substitutions and rename it from the DATA_FILE name
     to ECLBASE
     """
-    if packaging.version.parse(version) > packaging.version.parse("2.43"):
+    if packaging.version.parse(version) > packaging.version.parse("4.0"):
         pytest.fail("DATA_FILE has passed deprecation period and should be removed")
     config_text = dedent(
         """

